src/ServiceControl.Audit.AcceptanceTests/Auditing/FailedAuditsController.cs

```diff
@@ -46,8 +38,9 @@ public async Task<HttpResponseMessage> ImportFailedAudits(CancellationToken canc
             return Request.CreateResponse(HttpStatusCode.OK);
         }
 
-        readonly IDocumentStore store;
+        //readonly IDocumentStore store;
```

Comment:
Presumably this can be droppped as it's not used

Reply:
Agreed, will fix on the raven branch